collie/interactions/datasets.py

```diff
@@ -67,15 +70,15 @@ class BaseInteractions(torch.utils.data.Dataset, metaclass=ABCMeta):
         Number of items in the dataset. If ``num_items == 'infer'``, this will be set to the
         ``mat.shape[1]`` or ``max(items) + 1``, depending on the input
     check_num_negative_samples_is_valid: bool
-        Check that ``num_negative_samples`` is less than the maximum number of items a user has
-        interacted with. If it is not, then for all users who have fewer than
-        ``num_negative_samples`` items not interacted with, a random sample including positive items
-        will be returned as negative
+        Check that ``num_negative_samples`` is less than the maximum number of interactions between
+        users and items depending on ``negative_sample_type``. If it is not, then for all users or
+        items who have fewer interactions than ``num_negative_samples``, a random sample including
+        positive ids will be returned as negative
```

Comment:
[Nitpicky]

```suggestion
        positive IDs will be returned as negative
```

Reply:
Same for the rest of this file where this appears.

Reply:
Whoops! I know you called this out before, apologies for the double reminder!